website/docs/r/lexv2models_bot.html.markdown

```diff
@@ -0,0 +1,86 @@
+---
+subcategory: "Lex V2 Models"
+layout: "aws"
+page_title: "AWS: aws_lexv2models_bot"
+description: |-
+  Terraform resource for managing an AWS Lex V2 Models Bot.
+---
+
+# Resource: aws_lexv2models_bot
+
+Terraform resource for managing an AWS Lex V2 Models Bot.
+
+## Example Usage
+
+### Basic Usage
+
+```terraform
+resource "aws_lexv2models_bot" "example" {
+  name = "example"
+  data_privacy {
+    child_directed = "boolean"
+  }
+  idle_session_ttl_in_seconds = 10
+  role_arn                    = "bot_example_arn"
+}
+```
+
+## Argument Reference
+
+The following arguments are required:
+
+* `name` - The name of the bot. The bot name must be unique in the account that creates the bot. Type String. Length Constraints: Minimum length of 1. Maximum length of 100.
+* `data_privacy` - Provides information on additional privacy protections Amazon Lex should use with the bot's data. See [`data_privacy`](#data-privacy)
+* `idle_session_ttl_in_seconds` - The time, in seconds, that Amazon Lex should keep information about a user's conversation with the bot. You can specify between 60 (1 minute) and 86,400 (24 hours) seconds.
```

Comment:
```suggestion
* `idle_session_ttl_in_seconds` - Time, in seconds, that Amazon Lex should keep information about a user's conversation with the bot. You can specify between 60 (1 minute) and 86,400 (24 hours) seconds.
```

Convention (not always followed very well) is to drop "The", "A", and "An" at the beginning of argument descriptions.